src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -7,21 +7,69 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Reflection;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.Metadata;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using System.Runtime.CompilerServices;
 
 namespace System.Reflection.Emit.Experimental
 {
+
     public class AssemblyBuilder: System.Reflection.Assembly
     {
-        internal AssemblyBuilder() { }
+        private static readonly Guid _s_guid = new Guid("87D4DBE1-1143-4FAD-AAB3-1001F92068E6");//Some random ID, Need to look into how these should be generated.
```

Comment:
```suggestion
        private static readonly Guid s_guid = new Guid("87D4DBE1-1143-4FAD-AAB3-1001F92068E6");//Some random ID, Need to look into how these should be generated.
```
The prefix for statics is just `s_`